src/core/lib/security/credentials/ssl/ssl_credentials.c

```diff
@@ -157,6 +197,65 @@ static void ssl_build_server_config(
     config->pem_key_cert_pairs[i].private_key =
         gpr_strdup(pem_key_cert_pairs[i].private_key);
   }
+
+  return config;
+}
+
+void grpc_ssl_server_certificate_config_release(
+    grpc_ssl_server_certificate_config *config) {
+  size_t i;
+
+  if (config == NULL) return;
+
+  for (i = 0; i < config->num_key_cert_pairs; i++) {
+    gpr_free((void *)config->pem_key_cert_pairs[i].private_key);
```

Comment:
nit: I don't think that you need the `(void *)` cast in C.

Reply:
This is to cast away the constness of `private_key`.